packages/mediapipe-task-text/build.dart

```diff
@@ -1 +1,57 @@
-void main(List<String> args) async {}
+import 'dart:io';
+import 'package:http/http.dart' as http;
+import 'package:native_assets_cli/native_assets_cli.dart';
+import 'package:native_toolchain_c/native_toolchain_c.dart';
+
+const assetFilename = 'libtext_classifier.dylib';
+const assetLocation =
+    'https://storage.cloud.google.com/random-storage-asdf/$assetFilename';
+
+File outputFile = File(
+    '/Users/craiglabenz/Dev/git/google/flutter-mediapipe/packages/mediapipe-task-text/logs-build.txt');
+
+final logs = <String>[];
+
+void main(List<String> args) async {
+  if (await outputFile.exists()) {
+    await outputFile.delete();
+  }
+  await outputFile.create();
+
+  log(args.join(' '));
+  _build(args);
+  outputFile.writeAsString(logs.join('\n'));
+}
+
+Future<void> _build(List<String> args) async {
+  final buildConfig = await BuildConfig.fromArgs(args);
+  final buildOutput = BuildOutput();
+  final downloadUri = Uri.parse(assetLocation);
+  final downloadFileLocation = buildConfig.outDir.resolve(assetFilename);
+  log('Downloading $downloadUri');
+  final downloadResponse = await http.get(downloadUri);
+  final downloadedFile = File(downloadFileLocation.toFilePath());
+  if (downloadResponse.statusCode == 200) {
+    if (downloadedFile.existsSync()) {
+      downloadedFile.deleteSync();
+    }
+    downloadedFile.createSync();
+    downloadedFile.writeAsBytes(downloadResponse.bodyBytes);
+  }
+  buildOutput.dependencies.dependencies
+      .add(buildConfig.packageRoot.resolve('build.dart'));
+  buildOutput.assets.add(
+    Asset(
+      id: 'package:mediapipe_text/src/mediapipe_text_bindings.dart',
```

Comment:
Couldn't work out exactly the purpose of this `id` parameter and whether or not this is appropriate

Reply:
It is the library uri where your `@Native` annotations are. (Assuming you don't specify a custom `assetId` in your `@Native` annotations.)

It would be packages/mediapipe-core/lib/src/third_party/mediapipe/generated/mediapipe_common_bindings.dart -> `package:mediapipe-core:src/third_party/mediapipe/generated/mediapipe_common_bindings.dart` in the other PR. (Though I see there are no functions at all in that file, just structs.)